ADOPTERS.md

```diff
@@ -1,5 +1,7 @@
- To add your organization to this list, simply add your organization's name,
- optionally with a link. The list is in alphabetical order.
+ To add your organization to this list, open a pull request that adds your
+ organization's name, optionally with a link. The list is in alphabetical order.
+
+ (Remember to use `git commit --signoff` to comply with the DCO)
 
 # Organizations Using Helm
```

Comment:
@technosophos Can the list be sorted in alphabetical order as mentioned in the doc?